registry/storage/driver/s3-aws/s3.go

```diff
@@ -105,25 +106,12 @@ type DriverParameters struct {
 }
 
 func init() {
-	for _, region := range []string{
-		"us-east-1",
-		"us-east-2",
-		"us-west-1",
-		"us-west-2",
-		"eu-west-1",
-		"eu-west-2",
-		"eu-central-1",
-		"ap-south-1",
-		"ap-southeast-1",
-		"ap-southeast-2",
-		"ap-northeast-1",
-		"ap-northeast-2",
-		"sa-east-1",
-		"cn-north-1",
-		"us-gov-west-1",
-		"ca-central-1",
-	} {
-		validRegions[region] = struct{}{}
+	resolver := endpoints.DefaultResolver()
+	partitions := resolver.(endpoints.EnumPartitions).Partitions()
```

Comment:
I see there is also a `endpoints.DefaultPartitions()`, would this always return the same list?

Reply:
Yes let's use that instead